index.html

```diff
@@ -268,6 +268,72 @@ <h4 class="experience-title">Skills</h4>
         </li>
       </ul>
     </section>
+    <section class="contact-form">
+      <p>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </p>
+      <form class="form" action="https://formspree.io/f/xknkyppy" method="POST">
+        <input
+          type="text"
+          name="name"
+          maxlength="30"
+          placeholder="Full name"
+          required
+        />
+        <input type="text" name="email" placeholder="Email address" required />
```

Comment:
- [ ] Please change the type of this input field to email just as required in the project requirements. Below is a quote for the related requirement:

  > All fields have the correct type.